core/src/main/scala/org/apache/spark/ui/jobs/AllJobsPage.scala

```diff
@@ -42,6 +42,18 @@ private[ui] class AllJobsPage(parent: JobsTab) extends WebUIPage("") {
     }
 
     def makeRow(job: JobUIData): Seq[Node] = {
+      // scalastyle:off
+      val killLink = if (killEnabled) {
+        val killLinkUri = "%s/jobs/job/kill?id=%s&terminate=true"
```

Comment:
Just a few tiny comments to consider, if you have to otherwise make changes to this PR: this might be easier to read with string interpolation